deployments/helm-chart/crds/transportserver.yaml

```diff
@@ -1,10 +1,11 @@
-{{- if .Values.controller.enableCustomResources }}
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: transportservers.k8s.nginx.org
   labels:
-    {{- include "nginx-ingress.labels" . | nindent 4 }}
+    app.kubernetes.io/name: "nginx-ingress"
+  annotations:
+    "helm.sh/hook": crd-install
```

Comment:
are there any differences in how to install CRDs for Helm 2 vs Helm 3? Or it will work the same - just this annotation is needed for helm 2.

Reply:
* Helm3 will install crds unless you set `helm install --skip-crds`.

* Helm2 will install crds unless you set `controller.enableCustomResources=false`

Aside from that the behaviour is the same. CRDs are installed before all other resources, are left behind after a release is removed for safety and must be upgraded via `kubectl`.